src/Exception/MalformedRequestBodyException.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @see       https://github.com/zendframework/zend-expressive for the canonical source repository
+ * @copyright Copyright (c) 2015 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   https://github.com/zendframework/zend-expressive/blob/master/LICENSE.md New BSD License
+ */
+
+namespace Zend\Expressive\Helper\Exception;
+
+use InvalidArgumentException;
+use Interop\Container\Exception\ContainerException;
+
+class MalformedRequestBodyException extends InvalidArgumentException implements
+    ContainerException,
```

Comment:
ContainerException is not necessary here, as this happens during runtime, not when fetching a service from the container.